proto/engine/v1/execution_engine.proto

```diff
@@ -105,6 +105,45 @@ message ExecutionPayloadHeaderCapella {
 	bytes withdrawals_root  = 15 [(ethereum.eth.ext.ssz_size) = "32"]; // New in Capella.
 }
 
+message ExecutionPayloadDeneb {
+	bytes parent_hash           = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes fee_recipient         = 2 [(ethereum.eth.ext.ssz_size) = "20"];
+	bytes state_root            = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes receipts_root         = 4 [(ethereum.eth.ext.ssz_size) = "32"];
+	bytes logs_bloom            = 5 [(ethereum.eth.ext.ssz_size) = "256"];
```

Comment:
This is spec configurable, should we hard code it here?

Reply:
13b966725fb78420f8d2a70412952d365942a15e

Reply:
Logs bloom can't be configured because it's a prefix. It's `ssz_size` not `ssz_max` so using variable fails here. I'm going to stick with 256 for now
https://buildkite.com/prysmatic-labs/prysm/builds/73059#0187ecda-dbca-4ee0-9c73-ff793e63409e